### ISSUE TITLE
Declare methods ignoring lists of already declared objects

### ISSUE THREAD

**Issue by kookynerd** [vyuldashev/laravel-queue-rabbitmq#315]
- Laravel Version: 6.14.0
- Package Version: 10.1.3
- RabbitMQ Version: 3.8.2

**Describe the bug**
Declare methods
`\VladimirYuldashev\LaravelQueueRabbitMQ\Queue\RabbitMQQueue::declareExchange`
`\VladimirYuldashev\LaravelQueueRabbitMQ\Queue\RabbitMQQueue::declareQueue`
are ignoring already declared objects, and re-declare them again with default arguments values each call.

**Steps To Reproduce**
```
$queueConnection = Illuminate\Support\Facades\Queue::connection(self::RMQ_CONNECTION_NAME);

$queueConnection->declareExchange(self::QUEUE_NAME, AMQPExchangeType::DIRECT, false);
$queueConnection->declareQueue(self::QUEUE_NAME, false, false);

//Raise exception if expected exchange and queue durability is false
$queueConnection->pushRaw($payload, self::QUEUE_NAME);
```

**Expected behavior**

```
$queueConnection = Illuminate\Support\Facades\Queue::connection(self::RMQ_CONNECTION_NAME);

$queueConnection->declareExchange(self::QUEUE_NAME, AMQPExchangeType::DIRECT, false);
$queueConnection->declareQueue(self::QUEUE_NAME, false, false);

//pushRaw should not re-declare exchange and queue
$queueConnection->pushRaw($payload, self::QUEUE_NAME);
```

**Comment by adm-bome** [comment]
@kookynerd 

Please try the master branch. A lot of issues are already resolved. but not yet released.

`composer require vyuldashev/laravel-queue-rabbitmq:dev-master`

**Comment by kookynerd** [comment]
@adm-bome
Unfortunately does not work, probably because
`\VladimirYuldashev\LaravelQueueRabbitMQ\Queue\RabbitMQQueue::$exchanges`
`\VladimirYuldashev\LaravelQueueRabbitMQ\Queue\RabbitMQQueue::$queues`
do not changing during run time, but the checks for this properties are presented in methods mentioned in issue description

**Comment by adm-bome** [comment]
@kookynerd, Will look into this. 
The current code (master) only declares an exchage or queue when not already on the rabbitMQ server. The properties `$exchange` and `$queues` in the class are indeed unused at the moment.

Can you describe the case your working on, and why u need this extra check?

**Comment by kookynerd** [comment]
@adm-bome 
Exact case is pre-declare queue with durable=false, because PHP app not control worker that subscribed to that queue, and worker declare queue with that option.

**Comment by adm-bome** [comment]
Laravel Version: 7.8.1
Package Version: 10.2.0
RabbitMQ Version: 3.8.3

I can't reproduce the issue.

In RabbitMQ, I predeclared a queue `test` with no durability.
Within Laravel, I published (dispatched) a TestJob onto the queue `test` **with no errors**.
Then I started a worker `php artisan queue:work --queue=test --sleep=5 --tries=4`, **without errors**.

I also declared a queue with the command `php artisan rabbitmq:queue-declare --durable=0 another-test`

Same result.
